src/react/components/pages/editorPage/editorFooter.test.tsx

```diff
@@ -68,36 +65,36 @@ describe("Footer Component", () => {
         expect(tags[0].color).toEqual(originalTags[0].color);
     });
 
-    it("clicking 'ok' in modal closes and calls onChangeHandler", () => {
-        const onChangeHandler = jest.fn();
-        const wrapper = createComponent({
-            tags: originalTags,
-            displayHotKeys: true,
-            onTagsChanged: onChangeHandler,
-        });
-        wrapper.find("div.tag")
-            .first()
-            .simulate("click", { target: { innerText: originalTags[0].name }, ctrlKey: true});
-        wrapper.find("button.btn.btn-success").simulate("click");
-        expect(onChangeHandler).toBeCalled();
-    });
-
-    it("clicking 'cancel' in modal closes and does not call onChangeHandler", () => {
-        const onChangeHandler = jest.fn();
-        const wrapper = createComponent({
-            tags: originalTags,
-            displayHotKeys: true,
-            onTagsChanged: onChangeHandler,
-        });
-        wrapper.find("div.tag")
-            .first()
-            .simulate("click", { target: { innerText: originalTags[0].name }, ctrlKey: true});
-        wrapper.find("button.btn.btn-secondary").simulate("click");
-
-        expect(onChangeHandler).not.toBeCalled();
-    });
-
-    it("clicking tag without ctrl calls onTagClickHandler ", () => {
+    // it("clicking 'ok' in modal closes and calls onChangeHandler", () => {
```

Comment:
Why are these tests commented out?  Please uncomment and ensure they are passing.

Reply:
These are edited out because they are duplicated in tests on the Tags Input component in the vott-react repo

Reply:
They should still be included. It's more of an integration test at this point, making sure the footer is able to interact with the tags input component and pass along the appropriate data to the next level up

Reply:
Also, with the refactor, `TagsInput` no longer contains the editor modal. So any tests that are testing the actual editing of tags through the modal would need to be done here and the project form

Reply:
Now passing